src/architecture/SF2.0-basics.seq

```diff
@@ -0,0 +1,46 @@
+seqdiag {
+  # Change edge metrics
+  edge_length = 150; // default value is 192
+  span_height = 0;   // default value is 40
+
+  # Change note color
+  default_note_color = lightblue;
+
+  # actors (in order left to right)
+  Consumer; Broker; APIServer; Manager; Service;
+
+  # trigger provisioning of service instance
+  Consumer => Broker [label = "Create/Update/Delete", return = "ok"] {
+    Broker => APIServer [label = "Create resource with state as in_queue/update/delete", return = "true"];
```

Comment:
state will just be `in_queue` , correct ?

Reply:
This diagram presents all the CRUD operations.
For create, the state is `in_queue`, for update, it is `update` and for delete, it is `delete`